__tests__/app.test.js

```diff
@@ -185,6 +201,51 @@ describe("/api/articles/:article_id/comments", () => {
         expect(message).toBe("id not found");
       });
   });
+
+  test("POST 400: Respond with an error when passed an article_id with an incorrect format", () => {
+    const postComment = {
+      username: "butter_bridge",
+      body: "test-body",
+    };
+    return request(app)
+      .post("/api/articles/invalid_id_format/comments")
+      .send(postComment)
+      .expect(400)
+      .then(({ body }) => {
+        const { message } = body;
+        expect(message).toBe("invalid id type");
+      });
+  });
```

Comment:
I would add another 400 test where the request body is not in the right format, Ie username: [1,2,3] or userblame : butter_bridge 👍